.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,56 @@
+---
+name: Bug Report
+about: Report a bug or unexpected behavior in Macaron.
+title: "[Bug] - [Describe Issue]"
+labels: bug, triage
+assignees: ''
+---
+
+### Description
+Please provide a clear and concise description of the issue you're experiencing with Macaron. Be as detailed as possible about the problem.
+
+### Steps to Reproduce
+Please list the steps required to reproduce the issue:
+
+1. **Step 1**: [Describe the first step]
+2. **Step 2**: [Describe the second step]
+3. **Step 3**: [Describe the third step]
+4. [Continue adding steps if necessary]
+
+### Expected Behavior
+What were you expecting to happen?
+
+### Actual Behavior
+What actually happened? Please include any error messages, logs, or unexpected behavior you observed.
+
+### Debug Information
+Please run the command again with the `--verbose` option to provide debug information. This will help us diagnose the issue more effectively. You can add this option to the command like this:
+
+```shell
+./run_macaron.sh --verbose [other options]
+```
+
+Attach the debug output here if possible.
+
+### Environment Information
+To assist with troubleshooting, please provide the following information about your environment:
+
```

Comment:
Should we require CPU architecture information here so that it's a bit more explicit ?

Reply:
See commit [8489639](https://github.com/oracle/macaron/pull/1040/commits/84896396deb408a3cebfa3f7228ed579721eefc7).